playlist_collection/m3u_to_playlist.py

```diff
@@ -8,6 +8,15 @@
 	requests
 Setup:
 	Fill the variables below firstly, then run the script with -h to see the arguments that you need to give.
+
+Use with relative file path M3U files:
+	- The script assumes the .m3u file is placed at the root of the music folder.
+	- open the terminal from the folder with your music (and where the .m3u file is placed)
+	- run the script with the additional argument "--LibraryPath" including the path of the files within Plex to your music library.
+	Example:
+	linux: python3 ./m3u_to_playlist.py -l "Music" -f "chill.m3u" -u @all -p /media/Music/
```

Comment:
I would advice to put quotes around the `/media/Music` paths in both examples. As people will take over the examples and fill their own paths in, the example will fall apart when the filepath contains spaces. So just replace it with `... -p "/media/Music"` for both examples.